src/main/isNotEmail.ts

```diff
@@ -0,0 +1,33 @@
+import {
+  registerDecorator,
+  ValidationOptions,
+  ValidatorConstraint,
+  ValidatorConstraintInterface,
+  ValidationArguments
+} from '@hmcts/class-validator'
+
+import * as validator from 'validator'
+
+@ValidatorConstraint()
+export class IsNotEmailConstraint implements ValidatorConstraintInterface {
+
+  validate (value: any, args: ValidationArguments): boolean {
+    return (typeof value === 'string' && value.length !== 0 && !validator.isEmail(value)) || value.length === 0
```

Comment:
The `&& value.length !== 0` is made redundant by the `value.length === 0` returning true on the other side of the or, could be simplified to:
`return value.length === 0 || (typeof value === 'string' && !validator.isEmail(value))`